soccer/gameplay/generate_fsm_diagrams.py

```diff
@@ -21,8 +20,20 @@ def mkdir_p(path):
 logging.getLogger().setLevel(logging.INFO)
 
 import main
+import ui.main
 main.init()
 
+class moc_Point():
```

Comment:
If you `import robocup` at the top of the file, you can use `robocup.Point` rather than creating a mock class here. Probably the same with the Ball mock too.